modules/nf-core/plink2/hwe/meta.yml

```
@@ -0,0 +1,131 @@
+---
+name: "plink2_hwe"
+description: Filters plink bfiles or pfiles with maf filters
+keywords:
+  - plink2
+  - filter
+  - samples
+  - variants
+  - hwe
+  - qualty
+tools:
+  - "plink2":
+      description: |
+        Whole genome association analysis toolset, designed to perform a range
+        of basic, large-scale analyses in a computationally efficient manner
+      homepage: http://www.cog-genomics.org/plink/2.0/
+      documentation: http://www.cog-genomics.org/plink/2.0/general_usage
+      doi: "10.1186/s13742-015-0047-8"
+      licence: ["GPL v3"]
+      identifier: ""
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'test' ]
+    - plink_genotype_file:
+        type: file
+        description: PLINK binary genotype table file or PLINK 2 binary genotype table file
+        pattern: "*.{bed,pgen}"
+        ontologies:
+          - edam: http://edamontology.org/format_3003
+    - plink_variant_file:
+        type: file
+        description: PLINK extended MAP file or PLINK 2 variant information file
+        pattern: "*.{bim,pvar}"
+    - plink_sample_file:
+        type: file
+        description: PLINK sample information file or PLINK 2 sample information file
+        pattern: "*.{fam,psam}"
+  - - hwe:
+        type: float
+        description: Threshold to use for maf filtering
+
+output:
+  - bed:
+      - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. [ id:'test', single_end:false ]
```

```suggestion
            e.g. [ id:'test', single_end:false ]
```
you could remove the single_end when its not applicable :)